parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveComparator.java

```diff
@@ -183,10 +183,10 @@ public String toString() {
   private static abstract class BinaryComparator extends PrimitiveComparator<Binary> {
     @Override
     int compareNotNulls(Binary o1, Binary o2) {
-      return compare(o1.toByteBuffer(), o2.toByteBuffer());
+      return compareBinary(o1, o2);
     }
 
-    abstract int compare(ByteBuffer b1, ByteBuffer b2);
+    abstract int compareBinary(Binary b1, Binary b2);
```

Comment:
nit: Maybe `compareBinaries` would be a better naming

Reply:
The `Binary` refers to a class not an object, hence it feels more natural as `compareBinary`